rfps/near-and-filecoin.md

```diff
@@ -0,0 +1,62 @@
+# NEAR x Filecoin
+
+This is a Request for Proposals for projects interested in exploring new opportunities at the intersection of NEAR and Filecoin.
+
+[NEAR](https://near.org/) is a decentralized application platform built to bridge the users of today’s Internet to the blockchain-based web of the future. NEAR is a PoS Layer 1 blockchain platform, built with UX and DevX in mind. NEAR’s novel sharding mechanism, Nightshade, parallelizes computation on chain and automatically increases capacity when needed, resulting in theoretically infinite scalability.
+
+[Filecoin](https://filecoin.io/) is a decentralized peer-to-peer storage network, allowing anyone to store and validate large-scale data as part of a global, 8-exabyte (and growing) network. Under the hood, all Filecoin nodes use the InterPlanetary File System (IPFS), a content-addressed protocol for storing and sharing data.
+
+In July 2021, the [Filecoin-NEAR bridge](https://blog.textile.io/native-filecoin-storage-for-blockchains/) developed by Textile was launched, taking the first step to provide simple, permissionless storage for smart contracts on NEAR. 
```

Comment:
expand to include a little more technical detail (1-2 sentences) about what this bridge does.